webui/src/lib/components/repository/changes.jsx

```diff
@@ -144,13 +174,19 @@ export const ChangesTreeContainer = ({results, delimiter, uriNavigator,
                                          leftDiffRefID, rightDiffRefID, repo, reference, internalRefresh, prefix,
                                          getMore, loading, nextPage, setAfterUpdated, onNavigate, onRevert,
                                          changesTreeMessage}) => {
+    const { expandAllDirs, expandMode, toggleAllDirs, updateOpenedDir, registerDir, markDirAsManuallyToggled, tick } = useExpandCollapseDirs();
     if (results.length === 0) {
         return <div className="tree-container">
             <Alert variant="info">No changes</Alert>
         </div>
     } else {
         return <div className="tree-container">
             {changesTreeMessage && <div>{changesTreeMessage}</div>}
+                    <div className="mb-2">
```

Comment:
This takes some vertical space from the files browser, and I don't think it's important enough for this.
Please either place it at the right end of the `<Card.Header>`, or find a less central location than the current one.

And - if this can be turned into an [icon](https://forum.gdevelop.io/t/expand-collapse-all-events-groups-buttons-in-the-events-editor/45308), it would be much better.